stories/module-analytics.stories.js

```diff
@@ -66,54 +66,42 @@ function SetupWrap( { children } ) {
 }
 
 storiesOf( 'Analytics Module', module )
-	.add( 'Account Property Profile Select (none selected)', () => {
-		const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+	.add( 'Account Property Profile Select', () => {
 		const setupRegistry = ( { dispatch } ) => {
+			const accountA = {
+				id: '1000',
+				name: 'Account A',
+			};
+
+			const propertyA = {
+				id: 'UA-2000-1',
+				name: 'Property A',
+			};
+
+			const propertyB = {
+				id: 'UA-2001-1',
+				name: 'Property B',
+			};
+
+			const profileA = {
+				id: '3000',
+				name: 'Profile A',
+			};
```

Comment:
Here it makes sense to label these, but I find these single-letter differentiators a bit tough to scan.

Let's use more descriptive labels—`propertyOne` and `propertyTwo` works, or `firstProperty` and `secondProperty`. Equally `properties = [ {}, {} ]` would work 🙂 

I know it seems nitpicky, but these end up being equivalent to single-letter-variables like `e` or `a` because that's the only differentiator between the two variables, which isn't as easy-to-scan as my suggestions 😅

Reply:
Updated.